src/app/pages/loading/loading-page.component.html

```diff
@@ -0,0 +1,2 @@
+<ish-loading></ish-loading>
+<p style="height: 400px;"></p>
```

Comment:
If an empty block element is needed to somehow make some room a `<div>` is the better choice since it does not have such a specific semantic meaning as a `<p>` = paragraph.
The used `errorpage` wrapperClass is kind of hacky too, but we can use it for now until we implement a more advanced PWA loading screen. ;)

Reply:
Yes, let's do that later when we properly use the Service Worker for something _real_. 😄